src/Atmos/Model/AtmosModel.jl

```diff
@@ -569,7 +595,7 @@ Store Cartesian coordinate information in `aux.coord`.
 """ init_state_auxiliary!
 function init_state_auxiliary!(m::AtmosModel, aux::Vars, geom::LocalGeometry)
     aux.coord = geom.coord
-    atmos_init_aux!(m.orientation, m, aux, geom)
+    init_aux!(m.orientation, m.param_set, aux)
```

Comment:
Would this be an issue if other `Common` components export `init_aux(balancelaw.property, ...)`

Reply:
No, we'd just need to `import` (and export) it if we wanted to use the same function (and define new methods). Maybe we should change the name to `init_aux_orientation!`? Idk what's the best naming strategy here. There's also the question of whether we should pass in `aux` or `aux.orientation` and change `init_aux!` appropriately. @simonbyrne, thoughts? Since we'll likely apply similar patterns to `TurbulenceClosures` and `Hyperdiffusion` modules.

Reply:
Let's merge and we can change the name later.